### PR TITLE
net, refactor: Rename CNetMessage::m_command with CNetMessage::m_type

### PR DESCRIPTION
**Pull request by hebasto**
https://github.com/bitcoin/bitcoin/pull/18533#issue-594592488:
> a message is not a command, but simply a message of some type

Continuation of bitcoin/bitcoin#18533 and bitcoin/bitcoin#18937.

### CODE REVIEW

**Review by theStack — APPROVED** [review]
Concept and code-review ACK 224d87855ec38cc15866d9673e1b19942a82c1cd

There are still quite a few instances of the confusing "command" terminology for message types around, as `$ git grep -i command ./src/net*` shows. Happy to review further changes in this direction (either in this PR or a follow-up).

**Review by shaavan — APPROVED** [review]
Code Review ACK 224d87855ec38cc15866d9673e1b19942a82c1cd

Going commit wise:
1. I agree with this statement, and hence with the change:
> a message is not a command, but simply a message of some type

2. Removing these redundant variables makes the code more clean and readable.

**Review by w0xlt — APPROVED** [review]
crACK 224d878

File: src/net.cpp [hunk 1]
```diff
@@ -747,7 +747,7 @@ CNetMessage V1TransportDeserializer::GetMessage(const std::chrono::microseconds
     CNetMessage msg(std::move(vRecv));
 
     // store command string, time, and sizes
```
Comment (MarcoFalke) [inline src/net.cpp:749]: I think there are a few comments which could be updated.

```
git grep --ignore-case command ./src/net*

Comment (shaavan) [reply src/net.cpp:749]: Let me take it up as a follow-up.

**Pull request merged**